### DISCUSSION TITLE
TypeError: e.changeLanguage is not a function

### DISCUSSION THREAD

**Discussion by mustaqball** [sergiodxa/remix-i18next#111] [Q&A] [answered]
**app/root.jsx**

`import { useChangeLanguage } from 'remix-i18next';`

`const { locale } = useLoaderData();`

`useChangeLanguage(locale);`

This hook **useChangeLanguage** throws an error

**Comment by sergiodxa** [comment, marked as answer]
This has been reported here https://github.com/sergiodxa/remix-i18next/issues/107, I'm locking this conversation to keep the bug report on issues.
